lib/hope-tab.js

```diff
@@ -58,6 +59,47 @@ window.HopeTab = fxosComponent.register('hope-tab', {
     }
   },
 
+  onError(e) {
+    debug('on error', e, this.els.input.value, this.userSubmittedUrl);
+    var networkErrors = [
+      'dnsNotFound',
+      'connectionFailure'
```

Comment:
Yeah I was a little worried by https downgrade on navigation changes within
the tab. Assuming we can prove this isn't possible I'm happy to add this in.

On Wed, Feb 3, 2016 at 11:31 AM Etienne Segonzac <notifications@github.com>
wrote:

> In lib/hope-tab.js
> <https://github.com/kite-project/hope/pull/38#discussion_r51709510>:
>
> > @@ -58,6 +59,47 @@ window.HopeTab = fxosComponent.register('hope-tab', {
> >      }
> >    },
> >
> > +  onError(e) {
> > +    debug('on error', e, this.els.input.value, this.userSubmittedUrl);
> > +    var networkErrors = [
> > +      'dnsNotFound',
> > +      'connectionFailure'
>
> looks like we need to downgrade for certerror too
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kite-project/hope/pull/38/files#r51709510>.
>